Xamarin.Forms.ControlGallery.WindowsUniversal/CustomRenderers.cs

```diff
@@ -115,7 +115,9 @@ public static void SetKeyboardFlags(this FormsTextBox Control, KeyboardFlags? fl
 	}
 
 
+#pragma warning disable 618
 	public class TextBoxViewRenderer : BoxViewRenderer
```

Comment:
We can't use the new renderer here too? If it's obsolete, I guess we shouldn't be using it anymore either 😉

Reply:
Fine with me, updated